editor/src/clj/editor/engine/native_extensions.clj

```diff
@@ -150,11 +150,12 @@
 (defn get-build-server-url
   (^String [prefs project]
    (g/with-auto-evaluation-context evaluation-context
-     (get-build-server-url prefs project evaluation-context)))
+     (string/trim (get-build-server-url prefs project evaluation-context))))
```

Comment:
This code shouldn't be trimming since it uses 3-arity variant of the same `get-build-server-url` function that already trims the string.

Reply:
true. thanks